client/data/marketing-banner/use-marketing-banner.ts

```diff
@@ -1,24 +0,0 @@
-import { useQuery, UseQueryResult } from '@tanstack/react-query';
-import wpcom from 'calypso/lib/wp';
-
-export type MarketingBanner = {
-	is_visible: boolean;
-};
-
-export const useMarketingBanner = (
-	siteId: number,
-	isEnabled = false
-): UseQueryResult< MarketingBanner > => {
-	const queryKey = [ 'marketing-banner', siteId ];
-
-	return useQuery< MarketingBanner >( {
-		queryKey,
-		queryFn: () => {
-			return wpcom.req.get( {
-				path: `/sites/${ siteId }/marketing-banner`,
```

Comment:
We can remove this endpoint now (added in D156183-code), since it's no longer used.

Reply:
Done in D166376-code